django-stubs/db/models/query.pyi

```diff
@@ -98,8 +110,8 @@ class _QuerySet(Generic[_T, _Row], Collection[_Row], Reversible[_Row], Sized):
     async def aexists(self) -> bool: ...
     def explain(self, *, format: Optional[Any] = ..., **options: Any) -> str: ...
     async def aexplain(self, *, format: Optional[Any] = ..., **options: Any) -> str: ...
-    def contains(self, objs: Iterable[_T]) -> bool: ...
-    async def acontains(self, objs: Iterable[_T]) -> bool: ...
+    def contains(self, obj: models.Model) -> bool: ...
+    async def acontains(self, obj: models.Model) -> bool: ...
```

Comment:
using `_T` here gave me:

```
django-stubs/db/models/query.pyi:113: error: Cannot use a covariant type variable as a parameter
django-stubs/db/models/query.pyi:114: error: Cannot use a covariant type variable as a parameter
```

But looking at the source, it seems any instance of a model class is acceptable:

https://github.com/django/django/blob/b92ffebb0cdc469baaf1b8f0e72dddb069eb2fb4/django/db/models/query.py#L1241-L1255

hence change to `models.Model`